tests/queries/0_stateless/00921_datetime64_compatibility.reference

```diff
@@ -1293,98 +1292,99 @@ Code: 43: Illegal types of arguments (UInt64, Date) of function greater.
 "UInt8",0
 "UInt8",0
 ------------------------------------------
-SELECT toInt64(-1) == N
+SELECT N >= toInt8(-1)
 
-Code: 43: Illegal types of arguments (Int64, Date) of function equals.
+Code: 43: Illegal types of arguments (Date, Int8) of function greaterOrEquals.
+"UInt8",1
+"UInt8",1
+------------------------------------------
+SELECT toInt8(-1) >= N
+
+Code: 43: Illegal types of arguments (Int8, Date) of function greaterOrEquals.
 "UInt8",0
 "UInt8",0
 ------------------------------------------
-SELECT N != toInt64(-1)
+SELECT N >= toUInt16(1)
 
-Code: 43: Illegal types of arguments (Date, Int64) of function notEquals.
+Code: 43: Illegal types of arguments (Date, UInt16) of function greaterOrEquals.
 "UInt8",1
 "UInt8",1
 ------------------------------------------
-SELECT toInt64(-1) != N
+SELECT toUInt16(1) >= N
 
-Code: 43: Illegal types of arguments (Int64, Date) of function notEquals.
+Code: 43: Illegal types of arguments (UInt16, Date) of function greaterOrEquals.
+"UInt8",0
+"UInt8",0
+------------------------------------------
+SELECT N >= toInt16(-1)
+
+Code: 43: Illegal types of arguments (Date, Int16) of function greaterOrEquals.
 "UInt8",1
 "UInt8",1
 ------------------------------------------
-SELECT N <  toInt64(-1)
+SELECT toInt16(-1) >= N
 
-Code: 43: Illegal types of arguments (Date, Int64) of function less.
+Code: 43: Illegal types of arguments (Int16, Date) of function greaterOrEquals.
 "UInt8",0
 "UInt8",0
 ------------------------------------------
-SELECT toInt64(-1) <  N
+SELECT N >= toUInt32(1)
 
-Code: 43: Illegal types of arguments (Int64, Date) of function less.
+Code: 43: Illegal types of arguments (Date, UInt32) of function greaterOrEquals.
 "UInt8",1
 "UInt8",1
 ------------------------------------------
-SELECT N <= toInt64(-1)
+SELECT toUInt32(1) >= N
 
-Code: 43: Illegal types of arguments (Date, Int64) of function lessOrEquals.
+Code: 43: Illegal types of arguments (UInt32, Date) of function greaterOrEquals.
 "UInt8",0
 "UInt8",0
 ------------------------------------------
-SELECT toInt64(-1) <= N
+SELECT N >= toInt32(-1)
 
-Code: 43: Illegal types of arguments (Int64, Date) of function lessOrEquals.
+Code: 43: Illegal types of arguments (Date, Int32) of function greaterOrEquals.
 "UInt8",1
 "UInt8",1
 ------------------------------------------
-SELECT N >  toInt64(-1)
+SELECT toInt32(-1) >= N
 
-Code: 43: Illegal types of arguments (Date, Int64) of function greater.
+Code: 43: Illegal types of arguments (Int32, Date) of function greaterOrEquals.
+"UInt8",0
+"UInt8",0
+------------------------------------------
+SELECT N >= toUInt64(1)
+
+Code: 43: Illegal types of arguments (Date, UInt64) of function greaterOrEquals.
 "UInt8",1
 "UInt8",1
 ------------------------------------------
-SELECT toInt64(-1) >  N
+SELECT toUInt64(1) >= N
 
-Code: 43: Illegal types of arguments (Int64, Date) of function greater.
+Code: 43: Illegal types of arguments (UInt64, Date) of function greaterOrEquals.
```

Comment:
It is not trivial to check diff in this reference. I hope you've tried to check that nothing was broken or lost.

Reply:
> It is not trivial to check diff in this reference.

Indeed (test is tricky)

> I hope you've tried to check that nothing was broken or lost.

But yes, the problem basically is in the order, and it has been fixed separately in 5228c5b8aa3ea698e4050e960c84835dacb00e62:

    Fix order changes after converting to python3 in 00921_datetime64_compatibility.python
    
    - python3: {'op': ['-', '!=', '==', '< ', '<=', '> ', '>='], 'arg': ['DT', 'D', 'DT64']}
    - python2: {'arg': ['DT', 'D', 'DT64'], 'op': ['-', '!=', '==', '< ', '<=', '> ', '>=']}